docs/docs/feature-reference/adding-and-updating-custom-databases.md

```diff
@@ -97,9 +96,45 @@ tc.update_database(
 )
 ```
 
+Note that it may take up to 1 hour for the custom database to be ready to use.
```

Comment:
I'm not sure if this timing is true. Maybe something like "It takes time for the database to transfer to our system. Once the add / update run completes, you're good to go."